src/integration-tests/model-in-image/mii-sample-wrapper/build-model-image.sh

```diff
@@ -146,7 +200,12 @@ else
 
   tmpfil="$WORKDIR/$(basename $0).$CURPID.$PPID.$SECONDS.$RANDOM.sh"
 
-  output_dryrun | grep "^dryrun:" | sed 's/^dryrun://' > $tmpfil
+  if [ "${WDT_DOMAIN_TYPE}" = "JRF" ]; then
+    output_dryrun_JRF | grep "^dryrun:" | sed 's/^dryrun://' > $tmpfil
+  else
+    output_dryrun | grep "^dryrun:" | sed 's/^dryrun://' > $tmpfil
+  fi
+#  output_dryrun | grep "^dryrun:" | sed 's/^dryrun://' > $tmpfil
```

Comment:
There's a more compact way to do this than have have two versions of dryrun - see earlier comment.

Reply:
I agree. Thanks. Making the change now.